sdk/ai/azure-ai-projects/README.md

```diff
@@ -463,6 +553,81 @@ agent = await project_client.agents.create_agent(
 
 <!-- END SNIPPET -->
 
+#### Create Agent With Azure Function Call
+
+The agent can handle Azure Functions on the service side and return the result of the call. To use the function we need to create the `AzureFunctionTool`, which contains the input and output queues of azure function and the description of input parameters. Please note that in the prompt we are asking model to invoke queue when the specific question ("What would foo say?") is being asked. 
```

Comment:
"we are asking the model to invoke a queue"

Reply:
Fixed